index.html

```diff
@@ -0,0 +1,261 @@
+                    </p>
+                </div>
+            </div>
+        <!-- About Me Section -->
+        <section id="about-me" class="about-me">
+            <h2 class="section-heading">About Me</h2>
+            <div class="about-me-container">
+                <div class="about-me-image-container">
+                    <img src="./assets/images/interests-image.png" src="Josh playing guitar" class="about-me-image"/>
+                </div>
+                <div class="about-me-content">
+                    <h3>Personal Statement</h3>
+                    <p>
+                        I am a very hard-working and self-motivated individual who is keen to continue to learn and develop new skills.
+                    </p>
+                    <p>
+                        Working in the events industry has provided some great experiences, but I’m now looking for a new challenge in a career that is both sustainable and challenging while providing more opportunities for growth and development.
+                    </p>
+                    <p>
+                        I have spent time developing transferable skills including, communication, time management, organisation and many more. The key transferable skill is the ability to troubleshoot calmly and efficiently. Occasionally things go wrong during a show or production (especially when working with young students) and it’s crucial to be able to quickly get things back on track without anyone noticing.
+                    </p>
+                    <h3>Key Skills</h3>
+                    <ul>
+                        <li>Exceptional Communication</li>
+                        <li>Teamwork</li>
+                        <li>Fast and willing learner</li>
+                        <li>Proactive</li>
+                        <li>React Native, HTML, CSS, Python</li>
+                    </ul>
+                    <h3>Interests & Hobbies</h3>
+                    <p>
+                        I love having opportunities to challenge myself and be creative. I have been playing drums for over 10 years, and more recently picked up guitar and vocals. Since then I have been writing, recording and releasing my own music.
+                    </p>
+                    <p>
+                        Since starting this project in March 2021, I have released a few singles, played festivals and gigs, received support from the likes of BBC Introducing, and gained almost 80k streams on Spotify.
+                    </p>
+                    <p>
+                        I also enjoy keeping active, and play sports like tennis, squash, climbing and more recently, I’ve done my diving PADI which I want to explore more opportunities through that.
+                    </p>
+                </div>
+                
+            </div>
+        </section>
+
```

Comment:
There seems to be an issue with your code here - you can see this as the two closing `</section>` tags have the same indentation. Prettier would usually fix this, so I would say there is an issue somewhere. Have a look and see if there are any prettier errors in your VS code to help you fix this.

Reply:
Thanks! Just changed that